library/src/androidTest/java/org/xmtp/android/library/ConversationsTest.kt

```diff
@@ -137,4 +138,80 @@ class ConversationsTest {
 
         assertEquals(allMessages.size, 15)
     }
+
+    @Test
+    fun testLoadConvos() {
```

Comment:
Again don't want to merge this test just because the potential for variability in the wallet on dev. But appreciate you writing it and testing it.